scripts/libraryScreen.js

```diff
@@ -135,10 +135,13 @@ H5P.BranchingScenario.LibraryScreen = (function () {
       this.wrapper.style.minHeight = '';
       return;
     }
-
+    
```

Comment:
@ravimajithia The solution is OK, but please check your changes before committing to avoid adding whitespace changes like this.